### PR TITLE
Project Plan for 4.31

### PR DESCRIPTION
**Pull request by elsazac**
Project Plan created for 4.31.

cc/ @lshanmug

### CODE REVIEW

File: development/plans/eclipse_project_plan_4_31.xml [hunk 1]
```diff
@@ -0,0 +1,435 @@
...
+<!--
+NOTE: This is only necessary once per year. For us with the 202x-03 release.
+<p:milestone date="2023-11-25" milestone=""><div>Review Material Submission Deadline</div></p:milestone>
+-->
+<p:milestone date="2024-03-01" milestone="RC2"><div>4.31 RC2 - <b>New and Noteworthy Material Deadline</b></div></p:milestone>
+<p:milestone date="2024-03-13" milestone="GA"><div>4.31 GA</div></p:milestone>
+
+<p:postamble>
+<div>
+<p>Our target is to complete 4.31 in December 2023.
```
Comment (akurtakov) [inline development/plans/eclipse_project_plan_4_31.xml:117]: December 2023 is wrong.

Comment (elsazac) [reply development/plans/eclipse_project_plan_4_31.xml]: > December 2023 is wrong.

Addressed the change.

File: development/plans/eclipse_project_plan_4_31.xml [hunk 2]
```diff
@@ -0,0 +1,435 @@
...
+
+</div>
+</p:compatibility_with_previous_releases>
+  
+<!-- ============================================== -->
+
+<!--   <p:themes_and_priorities>  -->
+
+<p:preamble>
+We no longer group the plan items into separate themes as this is does not provide real value to our users.
```
Comment (HeikoKlare) [inline development/plans/eclipse_project_plan_4_31.xml:409]:
```suggestion
We no longer group the plan items into separate themes as this does not provide real value to our users.
```

File: development/plans/eclipse_project_plan_4_31.xml [hunk 3]
```diff
@@ -0,0 +1,435 @@
...
+<div>
+<p>Each plan item listed in the sub-project plans covers a feature or API that is
+  to be added to the Eclipse Project deliverables, or some aspect of the Eclipse Project that is
+  to be improved. Each plan item has its own entry in the Eclipse bugzilla database, with a title
```
Comment (HeikoKlare) [inline development/plans/eclipse_project_plan_4_31.xml:413]: There is no bugzilla database anymore, is it?

File: development/plans/eclipse_project_plan_4_31.xml [hunk 4]
```diff
@@ -0,0 +1,435 @@
...
+  throughout the release cycle. </p>
+<p>The first part of the plan deals with the important matters of release deliverables,
+  release milestones, target operating environments, and release-to-release compatibility.
+  These are all things that need to be clear for any release, even if no features
+  were to change. </p>
+<p>The remainder of the plan consists of plan items for all of the sub-projects
+  under the top level Eclipse Project. Each plan item covers a feature or API
+  that is to be added to the Eclipse Project deliverables, or some aspect of
+  the Eclipse Project that is to be improved. Each plan item has its own entry
+  in the Eclipse bugzilla database, with a title and a concise summary (usually
```
Comment (HeikoKlare) [inline development/plans/eclipse_project_plan_4_31.xml:47]: There is no bugzilla database anymore, is it?

Comment (elsazac) [reply development/plans/eclipse_project_plan_4_31.xml]: That was stale information carried over from the previous release.We don't use bugzilla anymore.I have removed the whole line.

**Comment by elsazac** [comment]
@akurtakov 

I have removed the lines regarding Plan items as those are not mentioned in the plan anymore.

**Comment by akurtakov** [comment]
Thanks!

**Pull request merged**